core/src/crds_gossip_pull.rs

```diff
@@ -242,10 +242,9 @@ impl CrdsGossipPull {
     ) -> Vec<(f32, &'a ContactInfo)> {
         let mut rng = rand::thread_rng();
         let active_cutoff = now.saturating_sub(PULL_ACTIVE_TIMEOUT_MS);
-        crds.table
-            .values()
+        crds.get_nodes()
```

Comment:
Can we just use the `get_nodes_contact_info()` you defined above function to avoid the `value.value.contact_info().unwrap();`?